engines/base/src/main/java/org/alfresco/transform/base/config/MTLSConfig.java

```diff
@@ -179,11 +184,23 @@ private HttpClient createHttpClientWithSslContext(SslContextBuilder sslContextBu
     }
 
     private RestTemplate createRestTemplateWithSslContext(SSLContextBuilder sslContextBuilder) throws NoSuchAlgorithmException, KeyManagementException {
-        SSLContext sslContext = sslContextBuilder.build();
-        SSLConnectionSocketFactory sslContextFactory = hostNameVerificationDisabled ? new SSLConnectionSocketFactory(sslContext, NoopHostnameVerifier.INSTANCE)
-                : new SSLConnectionSocketFactory(sslContext);
+        final SSLConnectionSocketFactoryBuilder sslConnectionSocketFactoryBuilder =
```

Comment:
Exact same change as above in similar context. We just have a deprecated copy of code and a new one.